examples/basic_bot.py

```diff
@@ -25,7 +25,7 @@ async def add(ctx, left: int, right: int):
     await ctx.send(left + right)
 
 @bot.command()
-async def roll(ctx, dice: str):
+async def roll(ctx: commands.Context, dice: str):
```

Comment:
Context should not be typehinted here

Reply:
It is typehinted everywhere else, remove it in every command then?

Reply:
Yeah